### PR TITLE
SOV-568: radio button component implementation

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-568
https://sovryn.atlassian.net/browse/SOV-710

### CODE REVIEW

**Comment by Rickk137** [comment]
@pietro-maximoff could you add ``RadioButton`` and ``RadioButonGroup`` to ``2_molecules/index.tsx``

**Comment by Rickk137** [comment]
Screen Shot 2022-10-27 at 1 48 23 PM
disabled and name is not showing in the story book for QA to test

**Comment by Rickk137** [comment]
Screen Shot 2022-10-27 at 1 52 02 PM
The labelInfo doesn't get the disabled style

File: packages/ui/src/2_molecules/RadioButtonGroup/RadioButtonGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <legend>{label}</legend>
+    {options.map(
+      ({ label, name, disabled, labelInfo }: RadioButtonOption, index) => {
+        const id = `radio-option-${label}`;
+        return (
+          <RadioButton
+            id={id}
+            key={id}
+            label={label}
+            value={label}
```
Comment (Rickk137) [inline packages/ui/src/2_molecules/RadioButtonGroup/RadioButtonGroup.tsx]: I think you should add value to the ``RadioButtonOption`` and use it here

File: packages/ui/src/2_molecules/RadioButtonGroup/RadioButtonGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            name={name}
+            disabled={disabled}
+            defaultChecked={index === 0}
```
Comment (Rickk137) [inline packages/ui/src/2_molecules/RadioButtonGroup/RadioButtonGroup.tsx]: How about getting it as prop for RadioButtonGroupProps? (and if it was undefined none of the options be pre selected)

File: packages/ui/src/2_molecules/RadioButton/RadioButton.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+export default {
+  title: 'Molecule/RadioButton',
+  component: RadioButton,
+};
+
+const Template: Story<ComponentProps<typeof RadioButton>> = args => (
+  <>
+    <RadioButton {...args} />
+  </>
```
Comment (Rickk137) [inline packages/ui/src/2_molecules/RadioButton/RadioButton.stories.tsx]: ``` <RadioButton {...args} />```

**Review by Rickk137 — CHANGES_REQUESTED** [review]

**Review by creed-victor — APPROVED** [review]

**Comment by pietro-maximoff** [comment]
@soulBit thanks for checking, and regarding the legend tag I think semantically it would be correct to use it instead of the h3 tag, h3 is mainly used for content as I know. do you think it's better to leave it with h3?

**Review by creed-victor — APPROVED** [review]

**Review by soulBit — APPROVED** [review]

**Review by LunaSov — APPROVED** [review]

**Pull request merged**